daemon/execdriver/driver.go

```diff
@@ -93,6 +93,11 @@ type Pid struct {
 	HostPid bool `json:"host_pid"`
 }
 
+// UTS settings of the container
+type Uts struct {
```

Comment:
Style: this should be UTS. Please see https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Initialisms.